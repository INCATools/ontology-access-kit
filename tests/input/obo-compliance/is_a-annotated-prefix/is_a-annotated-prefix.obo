format-version: 1.4
ontology: is_a-annotated-prefix
idspace: dcterms http://purl.org/dc/terms/

[Term]
id: X:1
is_a: Y:1 {dcterms:source="PMID:123456"}

