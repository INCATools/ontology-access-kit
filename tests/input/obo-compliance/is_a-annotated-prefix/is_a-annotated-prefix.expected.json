{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/is_a-annotated-prefix.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    } ],
    "edges" : [ {
      "sub" : "http://purl.obolibrary.org/obo/X_1",
      "pred" : "is_a",
      "obj" : "http://purl.obolibrary.org/obo/Y_1",
      "meta" : {
        "basicPropertyValues" : [ {
          "pred" : "http://purl.org/dc/terms/source",
          "val" : "PMID:123456"
        } ]
      }
    } ]
  } ]
}